templates/404.json

```
@@ -2,12 +2,58 @@
   "page": [
     {
       "name": "irving/body-wrapper",
+      "config": {
+        "class_name": "site_main"
+      },
       "children": [
         {
-          "name": "irving/text",
+          "name": "irving/container",
           "config": {
-            "content": "templates/404.json"
-          }
+            "tag": "header",
+            "class_name": "page-header alignwide"
+          },
+          "children": [
+            {
+              "name": "irving/text",
+              "config": {
+                "tag": "h1",
+                "class_name": "page-title",
+                "content": "Nothing here"
+              }
+            }
+          ]
+        },
+        {
+          "name": "irving/container",
+          "config": {
+            "tag": "div",
+            "class_name": "error-404 not-found default max-width"
```

I think you missed a dash in the third classname:

```suggestion
            "class_name": "error-404 not-found default-max-width"
```